src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -836,7 +836,7 @@ class OperatorStore extends BaseStore {
    */
   unselectOperatorByPublicKey(operator: any) {
     Object.keys(this.selectedOperators).forEach((index) => {
-      if (this.selectedOperators[index].address + this.selectedOperators[index].id === operator.address + operator.id) {
+      if (`${this.selectedOperators[index].address}${this.selectedOperators[index].id}` === `${operator.address}${operator.id}`) {
```

Comment:
if you take a look at api response, there is no `address` anymore 
https://api.stage.ssv.network/api/v4/prater/operators

Reply:
done